package.json

```diff
@@ -14,28 +14,20 @@
     "node": ">=18.0.0"
   },
   "description": "Utilities to use in your style dictionary config",
-  "types": "dist/index.d.ts",
-  "module": "dist/index.js",
-  "main": "dist-cjs/index.js",
   "exports": {
-    ".": {
-      "import": "./dist/index.js",
-      "types": "./dist/index.d.ts",
-      "require": "./dist-cjs/index.js"
-    }
+    ".": "./dist/index.js"
```

Comment:
This can be simplified now that we don't have CJS dual publish, and typescript is smart enough afaik to find .d.ts files next to .js files if they're co-located, so can be simplified into a one-liner now